bugsnag/src/main/java/com/bugsnag/OnSession.java

```diff
@@ -0,0 +1,5 @@
+package com.bugsnag;
+
+interface OnSession {
```

Comment:
I think it might be worth annotating this with `@FunctionalInterface` as a "SAM" guarantee:

```suggestion
@FunctionalInterface
interface OnSession {
```